crates/aranya-runtime/src/testing/dsl.rs

```diff
@@ -679,6 +691,26 @@ where
                 assert_eq!(max_cut, command.max_cut()?);
             }
             TestRule::IgnoreExpectations { ignore } => sink.ignore_expectations(ignore),
+            TestRule::VerifyGraphIds { client, ref ids } => {
+                let mut state = clients
+                    .get(&client)
+                    .ok_or(TestError::MissingClient)?
+                    .borrow_mut();
+
+                let actual_ids: BTreeSet<GraphId> = state
+                    .provider()
+                    .list_graph_ids()
+                    .unwrap()
+                    .map(Result::unwrap)
+                    .collect();
+
+                let expected_ids: BTreeSet<GraphId> = ids
+                    .iter()
+                    .map(|id| graphs.get(id).unwrap().clone())
```

Comment:
Clippy lint for clone and might as well use `[]` since unwrapping.
```suggestion
                    .map(|id| graphs[id])
```

Reply:
Huh, I tried that before and it complained but now it's fine